openedx_tagging/core/tagging/import_export/api.py

```diff
@@ -48,12 +48,32 @@
 
 from django.utils.translation import gettext as _
 
+from .. import api as taxonomy_api
 from ..models import TagImportTask, TagImportTaskState, Taxonomy
 from .exceptions import TagImportError
 from .import_plan import TagImportPlan, TagImportTask
 from .parsers import ParserFormat, get_parser
 
 
+def create_taxonomy_and_import_tags(
```

Comment:
See [note](https://github.com/openedx/openedx-learning/pull/112/files#diff-cf610e6b1cd41175eba13b86d368c8d680949a471fbb6d42aaa17e0a852a6764R75) below -- I don't think we need this API method here or on the platform PR.